src/js/components/featuredContent/articleCard/ArticleCard.jsx

```diff
@@ -5,49 +5,43 @@
 
 import React, { useState, useEffect } from 'react';
 import PropTypes, { oneOfType } from "prop-types";
-import { ShareIcon, CardContainer, CardHero, CardBody } from 'data-transparency-ui';
-import { useDispatch } from 'react-redux';
-import { handleShareOptionClick } from 'helpers/socialShare';
-import { mediumScreen } from 'dataMapping/shared/mobileBreakpoints';
+import { CardContainer, CardHero, CardBody } from 'data-transparency-ui';
+// import { handleShareOptionClick } from 'helpers/socialShare';
+// import { mediumScreen } from 'dataMapping/shared/mobileBreakpoints';
```

Comment:
i don't see a need to keep around the share code or the mobile breakpoints.  there aren't any plans to add a share button to the cards.